project/MicroService.scala

```diff
@@ -22,6 +22,8 @@ trait MicroService {
   lazy val plugins: Seq[Plugins] = Seq.empty
   lazy val playSettings: Seq[Setting[_]] = Seq.empty
 
+  val hmrcRepoHost = java.lang.System.getProperty("hmrc.repo.host", "https://nexus-dev.tax.service.gov.uk")
```

Comment:
Is this used if you're running the zap test locally?

Reply:
I don't think this is used at all atm, I will check and then remove.